admin_app/src/app/dashboard/components/Overview.tsx

```diff
@@ -18,26 +22,123 @@ interface OverviewProps {
 const Overview: React.FC<OverviewProps> = ({ timePeriod }) => {
   const { token } = useAuth();
   const [statCardData, setStatCardData] = React.useState<StatCardProps[]>([]);
+  const [heatmapData, setHeatmapData] = React.useState<ApexData[]>([]);
+  const [timeseriesData, setTimeseriesData] = React.useState<ApexData[]>([]);
+  const [topContentData, setTopContentData] = React.useState<TopContentData[]>(
+    [],
+  );
+
+  const heatmapOptions: ApexOptions = {
+    chart: {
+      id: "usage-heatmap",
+      width: "100%",
+      height: "100%",
+    },
+    dataLabels: {
+      enabled: false,
+    },
+    title: {
+      text: "Usage Frequency",
+      margin: 2,
+      offsetX: 6,
+      offsetY: 8,
+      floating: false,
+      style: {
+        fontSize: "16px",
+        fontWeight: "bold",
+        fontFamily: undefined,
+        color: "#263238",
+      },
+    },
+    colors: ["#008FFB"],
+    plotOptions: {
+      heatmap: {
+        radius: 1,
+      },
+    },
+  };
+
+  const timeseriesOptions: ApexOptions = {
+    chart: {
+      id: "usage-timeseries",
+      stacked: true,
+    },
+    dataLabels: {
+      enabled: false,
+    },
+    xaxis: {
+      type: "datetime",
+    },
+    legend: {
+      position: "top",
+      horizontalAlign: "left",
+    },
+    colors: ["#E91E63", "#546E7A", "#2E93FA"],
```

Comment:
Should we add these colors to `appColors` in case we want to reuse them in other section of the dashboard?

Reply:
Good idea.